Interfaces/Base.lproj/Preferences.xib

```diff
@@ -1538,8 +1538,19 @@ Warning: If Sequel Ace and the server do not have at least one cipher suite in c
                         <binding destination="117" name="value" keyPath="values.CustomQuerySoftIndentWidth" id="2205"/>
                     </connections>
                 </stepper>
+                <button fixedFrame="YES" translatesAutoresizingMaskIntoConstraints="NO" id="xIN-qP-ogl">
+                    <rect key="frame" x="18" y="18" width="213" height="18"/>
+                    <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMinY="YES"/>
+                    <buttonCell key="cell" type="check" title="Syntax highlighting" bezelStyle="regularSquare" imagePosition="left" alignment="left" state="on" inset="2" id="VHx-Tf-Rw2">
+                        <behavior key="behavior" changeContents="YES" doesNotDimImage="YES" lightByContents="YES"/>
+                        <font key="font" metaFont="system"/>
+                    </buttonCell>
+                    <connections>
+                        <binding destination="117" name="value" keyPath="values.CustomQueryEnableSyntaxHighlighting" id="kJw-7M-n52"/>
+                    </connections>
+                </button>
             </subviews>
-            <point key="canvasLocation" x="200" y="1295"/>
+            <point key="canvasLocation" x="200" y="1295.5"/>
```

Comment:
Could you move the checkbox to the right side?

Reply:
Perhaps above the auto-completion checkbox as well, I believe it makes a little more sense there